tools/kdc/kdc-api-server/server/dcos.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"github.com/dcos/client-go/dcos"
+	"net/url"
+	"os"
+	"strings"
+)
+
+/**
+ * Create a new DC/OS client instance, by authenticating against the cluster
+ * using the service account secret given by the environment
+ */
+func CreateDCOSClientFromEnvironment() (*dcos.APIClient, error) {
```

Comment:
This does not need to be exported, right?

Reply:
Well ... I kind of didn't want to restrict my exports in advance. I am exporting anything that makes sense in per-file scope, considering it to be a potential library in the future... If you think that's a serious issue, I could revise it.

Reply:
Since everything is in a single package here, please change everything to be private. It's much easier to relax visibility when needed than to try and tighten it up when there are already users of the API and we decide that we want to change things.